lib/code_corps/policy/github_app_installation.ex

```diff
@@ -6,8 +6,8 @@ defmodule CodeCorps.Policy.GithubAppInstallation do
 
   alias CodeCorps.{GithubAppInstallation, User}
 
-  def create?(%User{} = user, %Ecto.Changeset{} = changeset),
-    do: changeset |> get_project |> owned_by?(user)
+  def create?(%User{} = user, params),
+    do: params |> get_project |> owned_by?(user)
 
   def update?(%User{} = user, %GithubAppInstallation{} = github_app_installation),
```

Comment:
Seems like you pointed out a bug we weren't aware of. There github app installation endpoint should only support `:index`, `:show` and `:create` in the controller. That means we do not really need an `:update` policy here, so feel free to remove it as well as the associated tests.

Reply:
Ok, all changes should be live now, thanks @begedin !